mailet/base/src/test/java/org/apache/mailet/base/MailetUtilTest.java

```diff
@@ -37,41 +37,41 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void testGetInitParameterParameterIsTrue() {
-        assertTrue(getParameterValued("true", true));
-        assertTrue(getParameterValued("true", false));
-        assertTrue(getParameterValued("TRUE", true));
-        assertTrue(getParameterValued("TRUE", false));
-        assertTrue(getParameterValued("trUE", true));
-        assertTrue(getParameterValued("trUE", false));
+    public void getInitParameterShouldReturnTrueWhenIsValueTrue() {
+        assertThat(getParameterValued("true", true)).isTrue();
+        assertThat(getParameterValued("true", false)).isTrue();
+        assertThat(getParameterValued("TRUE", true)).isTrue();
```

Comment:
First, you should remove the 3 cases with true as default value.
Second, I know it's bothering but you should have 3 tests methods that describe what you are testing.